README.md

```diff
@@ -1 +1,156 @@
-# recipeApp
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
```

Comment:
- [ ] Great work so far but it would be nice if you fix the Run tests link because it is broken 😨 Kindly  fix this link or remove it if you don't have any dedicated section fro this link 🤔 to make your readme descriptive and professional 👍

Reply:
I removed them